stdlib/public/core/Misc.swift

```diff
@@ -103,6 +103,7 @@ public func _getMangledTypeName(_ type: Any.Type)
 /// Returns the mangled name for a given type.
 @available(SwiftStdlib 5.3, *)
 @_unavailableInEmbedded
+@_optimize(none)
```

Comment:
Guessing this is left over from debugging work.

Reply:
Thanks!